crates/node/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "targets": {
```

Comment:
Let's move this to package.json perhaps?

Reply:
done